client/src/components/vote/VoteModal.tsx

```diff
@@ -31,53 +34,95 @@ export default class VoteModal extends Component<IVoteModalProps, IVoteModalStat
+                  <h2
+                    style={{
+                      fontFamily: "Roboto",
+                      marginBottom: "5vh",
+                      marginTop: "5vh",
+                    }}
+                  >
+                    Please wait
+                  </h2>
                 </Col>
               </Row>
               <Row className="text-center">
                 <Col sm={12}>
-                  <Loader type="Grid" color="#00BFFF" height="10%" width="10%" />
+                  <Loader style={{ marginTop: "20vh" }} type="Grid" color="#00BFFF" height="30%" width="30%" />
                 </Col>
               </Row>
               <Row className="text-center">
                 <Col sm={12}>
-                  <h4>Your transaction is being processed</h4>
+                  <h2
+                    style={{
+                      fontFamily: "Roboto",
+                      marginBottom: "5vh",
+                      marginTop: "5vh",
+                      textAlign: "center",
+                    }}
+                  >
+                    Your transaction is being processed{" "}
+                  </h2>
                 </Col>
               </Row>
             </Fragment>
           ) : (
             <Fragment>
-              {this.props.voting.info.answers.map((answer, index) => {
-                return (
-                  <Row>
-                    <Col sm={10} style={{ paddingLeft: "60px" }}>
-                      <h4>{answer}</h4>
-                    </Col>
-                    <Col sm={2} style={{ paddingRight: "30px" }}>
-                      <Button
-                        className="pull-right"
-                        value={index}
-                        onClick={() => this.handleAnswerClick(index)}
-                        {...((this.props.voting.info.isPrivileged !== null && !this.props.voting.info.isPrivileged) ||
-                        this.props.voting.info.hasUserVoted
-                          ? { disabled: true }
-                          : null)}
-                        {...(this.props.chosenAnswer === index ? { active: true } : null)}
-                      >
-                        Pick answer #{index + 1}
-                      </Button>
-                    </Col>
-                  </Row>
-                );
-              })}
+              <Panel>
+                <Panel.Heading>
+                  <Panel.Title className="text-center" style={{ fontSize: "1.5em", fontWeight: "bold" }}>
+                    Voting info
+                  </Panel.Title>
+                </Panel.Heading>
+                <Panel.Body>
+                  <ControlLabel style={{ fontSize: "1.2em" }}>Question</ControlLabel>
+                  <p>{this.props.voting.info.question}</p>
+                  <ControlLabel style={{ fontSize: "1.2em" }}>Voting deadline</ControlLabel>
+                  <p>Voting possible until {covertTimestampToDate(this.props.voting.info.votingEndTime)}</p>
```

Comment:
```suggestion
                  <p>{covertTimestampToDate(this.props.voting.info.votingEndTime)}</p>
```
display plain date as voting deadline info

Reply:
This way it's unified with `<ResultsModal />`